task1/src/SearchForm.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+/*
+SearchForm Using React.PureComponent
+ */
+export class SearchForm extends React.PureComponent {
+    constructor() {
+        super();
+        this.state = {
+            keyword: '',
+            showResults: false
+        }
+    }
+
+    search = (e) => {
+        const value = e.target.value;
+        if (value === '') {
+            this.setState({ showResults: false });
+        }
+        this.setState({ keyword: e.target.value });
+    }
+
+    submit = () => {
+        this.setState({ showResults: true });
+    }
+
+    render() {
+        return <form className='search-form'>
+            <input type="text" className='search-field' id="search-field" onChange={this.search} value={this.state.keyword} />
```

Comment:
good practice to write jsx with new line for each prop in case props are more then 1-2.

something as:
```
<input 
  type="text"
  className='search-field'
  id="search-field"
  onChange={this.search}
  value={this.state.keyword}
  />
```

Reply:
updated